pkgs/applications/window-managers/dwm/default.nix

```diff
@@ -25,6 +25,10 @@ stdenv.mkDerivation rec {
         if lib.isDerivation conf || builtins.isPath conf
         then conf else writeText "config.def.h" conf;
     in
+    ''
+      substituteInPlace config.mk \
+        --replace "CC = cc" "CC = ${stdenv.cc.targetPrefix}cc"
+    '' +
     lib.optionalString (conf != null) "cp ${configFile} config.def.h";
 
```

Comment:
This should also work and does not require patching:

```
makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
```

Reply:
Oops, yes, that's better :). Thanks.